### PR TITLE
AIP 35: Adjust interest rate to account for APY over-approximation

### PR DESCRIPTION
**Pull request by ahnaguib**

### CODE REVIEW

File: content/aips/AIP-35.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+## Simple Summary
+
+- Update interest rate on AMPL market to account for over-approximation in compounded interest computation.
+- Make use of the exponential curve due to the over-approximation to set higher max APY.
+
+
+## Motivation
+
+In AAVEs MathUtils an over-approximation in the application of the interest rate for computing compounded interest was discovered.
```
Comment (brandoniles) [inline content/aips/AIP-35.md:19]: Lines 19 and 21 seem to duplicate each other. Only need one of them?

**Review by brandoniles — COMMENTED** [review]
LGTM, thanks Ahmed. Does it make sense to link to the snapshot results somewhere?

File: content/aips/AIP-35.md (hunk 1 above)
Comment (ahnaguib) [reply content/aips/AIP-35.md:19]: Good catch. Fixed.

**Comment by ahnaguib** [comment]
> LGTM, thanks Ahmed. Does it make sense to link to the snapshot results somewhere?

The discussion has link to the snapshot.  I prefer to keep it neater and not include more links.

**Pull request merged**